src/Symfony/Component/Security/Http/Firewall.php

```diff
@@ -38,17 +36,15 @@ class Firewall implements EventSubscriberInterface
 
     public function __construct(FirewallMapInterface $map, EventDispatcherInterface $dispatcher)
     {
-        // the type-hint will be updated to the "EventDispatcherInterface" from symfony/contracts in 5.0
-
         $this->map = $map;
         $this->dispatcher = $dispatcher;
         $this->exceptionListeners = new \SplObjectStorage();
     }
 
     /**
-     * @internal since Symfony 4.3
+     * @internal
```

Comment:
what about removing `@internal` on methods now? We don't have to keep it, do we?